references/plugins/references.go

```diff
@@ -416,7 +416,7 @@ func (ref *MarkdownReference) CreateMarkdown(ctx context.Context, caps []types.C
 				return err
 			}
 		case types.HelmCategory:
-			properties, _, err := ref.GenerateHELMProperties(ctx, &caps[i])
+			properties, _, err := ref.GenerateHelmAndKubeProperties(ctx, &caps[i])
```

Comment:
you may need to support `vela show --web` to show the  reference docs of `KUBE` type ComponentDefinition